### PR TITLE
Fix rendering of item form of actuators

### PR DESCRIPTION
**Pull request by liach**
**The Issue**
Acturator item forms only had base but not the arrows rendering.
 
**The Proposal**
Moved actuator model loader to be controlled by the block type
Now we have many models instead of one; we do not use block state from world any more as that cannot carry any info for item form.
Removed useless inventory and normal variants. They are not used.
Dumped useless actuator item class and override methods in the enum class itself (avoid making an inner class)
 
**Possible Side Effects**
Shouldn't affect anything outside of actuators.
 
**Alternatives**
Not good ones that I am aware of.

### CODE REVIEW

**Comment by liach** [comment]
This is the fixed screenshot.

File: src/main/java/mods/railcraft/client/render/models/resource/ActuatorModel.java [hunk 1]
```diff
@@ -92,70 +108,92 @@
...
+        public CompositeModel(IBakedModel baseModel, IBakedModel redFlagModel, IBakedModel whiteFlagModel) {
+            this.baseModel = baseModel;
+            this.redFlagModel = redFlagModel;
+            this.whiteFlagModel = whiteFlagModel;
         }
 
         @Override
         public List<BakedQuad> getQuads(@Nullable IBlockState state, @Nullable EnumFacing side, long rand) {
-            if (state instanceof IExtendedBlockState)
-                state = ((IExtendedBlockState) state).getClean();
```
Comment (CovertJaguar) [inline src/main/java/mods/railcraft/client/render/models/resource/ActuatorModel.java:151]: This is necessary, see #1116

**Comment by CovertJaguar** [comment]
Could we also get a first person screenshot of holding one in hand?

And you mention "lots of models" but I don't see any additional JSONs added. Am I misunderstanding what you mean?

**Review by CovertJaguar — CHANGES_REQUESTED** [review]

File: src/main/java/mods/railcraft/client/render/models/resource/ActuatorModel.java (hunk 1 above)
Comment (liach) [reply src/main/java/mods/railcraft/client/render/models/resource/ActuatorModel.java:151]: We no longer use the state here.

**Review by CovertJaguar — APPROVED** [review]

**Pull request merged**